internal/cluster/cluster.go

```diff
@@ -1058,6 +1057,15 @@ func (m *Manager) GenerateAdditionalManifests(ctx context.Context, cluster *comm
 			return errors.Wrapf(err, "Cluster %s - failed to generate manifests for vmware hosts", cluster.ID.String())
 		}
 	}
+
+	if err := m.manifestsGeneratorAPI.AddTelemeterManifest(ctx, log, cluster); err != nil {
+		return errors.Wrap(err, "failed to add telemeter manifest")
+	}
+
+	if err := m.rp.operatorsAPI.GenerateManifests(ctx, cluster); err != nil {
```

Comment:
Why did you move it (it used to be before AddDisableVmwareTunnelOffloading) ?
Are you sure the order doesn't matter?

Reply:
No special reason, just so they are grouped by API (operators manifests comes from a different API than network manifests).

This is just file generation no?! I don't think the order matter.

Reply:
I don't know what it does, so now we need to check...
In essence, if you don't have a reason to change something - don't.

Reply:
I will fix the order to fit the original order then. Thanks for the tip.